src/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -261,11 +281,6 @@ protected function appendAddressToDomElement(\DOMElement $creditor, CustomerCred
             $postalAddressData = array($postalAddressData);
         }
 
-        // Generate nodes for each address line.
```

Comment:
This modification is suspect for a problem. We are supporting specific file versions, not specific banks. This does not look right.